Model/Vote.php

```diff
@@ -95,7 +96,14 @@ public function isVoteValid(ExecutionContextInterface $context)
         if (!$this->checkValue($this->value)) {
             $message = 'A vote cannot have a 0 value';
             $propertyPath = $context->getPropertyPath() . '.value';
-            $context->addViolationAt($propertyPath, $message);
+
+            if (version_compare(Kernel::VERSION, '2.5.0', '<')) {
```

Comment:
Please use feature detection, not version detection:

- it is possible to install different versions of HttpKernel and Validator
- it is possible to use the old validator in Valdiator 2.5+ (this is even the default behavior for PHP 5.3.8 and below)